keyboards/eek/keymaps/ledtest/rules.mk

```
@@ -0,0 +1,20 @@
+
+BOOTMAGIC_ENABLE = lite      # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = no       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = yes        # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality on B7 by default
+RGBLIGHT_ENABLE = yes        # Enable keyboard RGB underglow
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+HD44780_ENABLE = no 		# Enable support for HD44780 based LCDs
+
+#OPT_DEFS += -DFLIPPED      # Enable if board is flipped to have eek! silk on top
+#RGB_MATRIX_ENABLE = WS2812 # Enable for pretty RGB matrix effects
```

The only thing changed here is, right? 
```suggestion
RGBLIGHT_ENABLE = yes        # Enable keyboard RGB underglow
RGB_MATRIX_ENABLE = no       # Enable for pretty RGB matrix effects
```

---

Yeah, that's the only thing. I just thought it would be nice to have an LED test map ready even though these are the "easy" LEDs is this okay?